watch-and-repack.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Relative paths to the web repositories
+PENUMBRA_ZONE_WEB_PATH="../../web"
+PRAX_REPO_PATH="."
+
+# Ensure 'pack-public.sh' is executable
+chmod +x "$PENUMBRA_ZONE_WEB_PATH/pack-public.sh"
+
+# Remove any existing tarballs from packages in penumbra-zone/web 
+cleanup_tarballs() {
+  find "$PENUMBRA_ZONE_WEB_PATH/packages" -name "*.tgz" -type f -delete
+}
+
+# Repack the packages in penumbra-zone/web
+repack() {
+  (cd "$PENUMBRA_ZONE_WEB_PATH" && ./pack-public.sh)
+}
+
+# Install dependencies in prax-wallet/web
+install_prax() {
+  (cd "$PRAX_REPO_PATH" && pnpm add -w ../../web/packages/*/penumbra-zone-*-*.tgz)
+}
+
+# Reload webpack
+reload_webpack() {
+  (cd "$PRAX_REPO_PATH" && pkill -f "webpack-dev-server")
```

Comment:
pkill is not directory constrained. in scripts you should only kill by known pid or job control.

it should be possible to signal webpack to reload without killing it

Reply:
I originally was trying to reload webpack, but changes only seemed to propagate when webpack was killed and then reloaded with pnpm dev

Reply:
pkill uses pattern matching under the hood, but not being directory constrained isn't relevant here. It's true that it will terminate any process matching the pattern regardless of the current directory, so if there are multiple webpack dev server instances running, `pkill -f "pnpm run dev"` will terminate all of them. 

I made the pattern unique,` webpack-prax-dev-server` to only terminate the dev server in prax.

Reply:
<img width="1072" alt="Screenshot 2024-06-24 at 3 00 43 PM" src="https://github.com/prax-wallet/web/assets/70081547/1a8cb635-5ae2-4db2-bc60-5afc34add8a8">

will work on the webpack because it currently seems to spawn a new webpack for every refresh, consuming my system memory?

Reply:
webpack should have a way to output a pid, or a way to instruct it to reload modules. and if that isn't feasible it should be wrapped in something that can. pkill is the wrong approach